integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/localdictionary/LocalDictionarySupportAlterTableTest.scala

```diff
@@ -822,17 +821,17 @@ class LocalDictionarySupportAlterTableTest extends QueryTest with BeforeAndAfter
     sql(
       """
         | CREATE TABLE local1(id int, name string, city string, age int,add string)
-        | STORED BY 'carbondata' tblproperties('local_dictionary_include'='city')
+        | STORED BY 'carbondata' tblproperties('local_dictionary_exclude'='city')
```

Comment:
change local_dictionary_exclude to local_dictionary_include as the name suggests